src/ApiTestCase.php

```diff
@@ -29,6 +29,9 @@
  */
 abstract class ApiTestCase extends WebTestCase
 {
+    const JSON = 'application/json';
```

Comment:
This constants are available in [MediaType](https://github.com/Lakion/ApiTestCase/blob/master/src/MediaTypes.php)